### PR TITLE
Add sample effort to get rodent data [patch]

### PR DESCRIPTION
**Pull request by ha0ye**
Mostly formatting changes
  * indentation
  * replacing '=' with '<-' for assignment
  * replacing single quotes with double quotes
  * using snake_case for variables

Some minor functional changes
  * `na_drop` and `zero_drop` are expected to be logical, tests changed accordingly
  * redoing some of the processing code to be more tidyverse-esque (e.g. using `dplyr::filter` instead of row indices)
  * removing redundant `filter_plots` call (might want to revisit the processing pipeline as its own issue someday)

### CODE REVIEW

File: R/IndividualData.R [hunk 1]
```diff
@@ -28,26 +28,24 @@
...
+    dplyr::select(period, month, day = day.x, year, treatment, plot, stake,
+                  species, sex, hfl, wgt, tag, ltag)
 
   #### use new moon number as time index if time == "newmoon" ----
-  rodents = add_time(rodents, data_tables$newmoons_table, time)
-
-  return(rodents)
-
+  add_time(rodents, data_tables$newmoons_table, time) %>%
+    return()
```
Comment (DAPPERstats) [inline R/IndividualData.R]: I thought `%>%` didn't work going into `return()`? Does this need to be changed, too?

File: R/RodentData.R [hunk 2]
```diff
@@ -1,117 +1,95 @@
...
-#'
-#' @param trapping trapping_table from Portal data (can be filtered)
-#' @param plots plots_table from Portal data
-#' 
-#' @return trapping table with sampled column removed and treatment column 
-#'   added 
-#'
-#' @export 
-#'
-join_plots_to_trapping <- function(trapping, plots){
```
Comment (DAPPERstats) [inline R/RodentData.R]: `join_plots_to_trapping` is removed here, but is still used in the codebase (e.g. line 223 within this script)... did it get moved to a different script or does it need to be reinstated here?

Comment (ha0ye) [reply R/RodentData.R]: Ah, I moved it into `data_processing.R` to be with the other `join_XX_to_YY` function definitions.

Comment (DAPPERstats) [reply R/RodentData.R]: oh! dang github had shrunk `data_processing.R` so i couldn't see the code in it, i'll check it out.

File: R/data_processing.R [hunk 3]
```diff
@@ -599,11 +621,12 @@ join_census_to_dates = function(census_table, date_table, plots_table){
...
-  quadrat_table = dplyr::right_join(quadrat_data, census_table,
-                                    by=c("year"="year","season"="season",
-                                         "plot"="plot","quadrat"="quadrat"))
-  return(quadrat_table)
+join_census_to_quadrats <- function(quadrat_data, census_table) {
+  quadrat_data %>%
+    dplyr::right_join(census_table,
+                      by = c(year = "year", season = "season",
+                             plot = "plot", quadrat = "quadrat")) %>%
+    return()
```
Comment (DAPPERstats) [inline R/data_processing.R]: here's one more `%>% return()`

**Pull request merged**